From bfefc8fa1449f802baab40be11638ff688f7826f Mon Sep 17 00:00:00 2001
From: Theo GOUREAU <theo.goureau-ext@st.com>
Date: Mon, 21 Oct 2024 10:07:11 +0200
Subject: [PATCH 1334/1695] dts: stm32: remove LTDC rotation from
 stm32mp215.dtsi

Remove LTDC rotation from stm32mp215.dtsi.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 7a89a395d52c057f876403ecbf473262363f3a82 dts: stm32: remove LTDC rotation from stm32mp215.dtsi]
Change-Id: I3f8ddce4b066ac6d3429deba1677634c60e22001
Signed-off-by: Theo GOUREAU <theo.goureau-ext@st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/412803
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Reviewed-by: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
Domain-Review: Yann GAUTIER <yann.gautier@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
---
 core/arch/arm/dts/stm32mp215.dtsi | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/core/arch/arm/dts/stm32mp215.dtsi b/core/arch/arm/dts/stm32mp215.dtsi
index a1495ed41..94e5dc2cd 100644
--- a/core/arch/arm/dts/stm32mp215.dtsi
+++ b/core/arch/arm/dts/stm32mp215.dtsi
@@ -26,9 +26,5 @@
 			access-controllers = <&rifsc STM32MP21_RIFSC_LTDC_L2_ID>;
 			access-controller-names = "l3";
 		};
-		rot {
-			access-controllers = <&rifsc STM32MP21_RIFSC_LTDC_ROT_ID>;
-			access-controller-names = "rot";
-		};
 	};
 };
-- 
2.47.2

